[live555] update to 2023.11.30

### Library name

live555

### New version number

2023.11.30

### Other information that may be useful (release notes, etc...)

PS D:\> vcpkg install live555
warning: In the September 2023 release, the default triplet for vcpkg libraries changed from x86-windows to the detected host triplet (x64-windows). For the old behavior, add --triplet x86-windows . To suppress this message, add --triplet x64-windows .
Computing installation plan...
The following packages will be built and installed:
    live555:x64-windows@2023-11-08
Detecting compiler hash for triplet x64-windows...
-- Automatically setting %HTTP(S)_PROXY% environment variables to "127.0.0.1:33561".
Restored 0 package(s) from C:\Users\qiaoxk\AppData\Local\vcpkg\archives in 148 us. Use --debug to see more details.
Installing 1/1 live555:x64-windows@2023-11-08...
Building live555:x64-windows@2023-11-08...
-- Note: live555 only supports static library linkage. Building static library.
-- Downloading http://live555.com/liveMedia/public/live.2023.11.08.tar.gz -> live.2023.11.08.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from D:\ProgramData\cdev\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: D:\ProgramData\cdev\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
Downloading http://live555.com/liveMedia/public/live.2023.11.08.tar.gz
error: Failed to download from mirror set
error: http://live555.com/liveMedia/public/live.2023.11.08.tar.gz: failed: status code 404
[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(1048):
[DEBUG] Time in subprocesses: 0us
[DEBUG] Time in parsing JSON: 11us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 1058us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 576 ms (536849us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):

      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:

      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.

      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.

      3. Your proxy's remote server is out of service.

      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.


Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  ports/live555/portfile.cmake:3 (vcpkg_download_distfile)
  scripts/ports.cmake:170 (include)


error: building live555:x64-windows failed with: BUILD_FAILED
Elapsed time to handle live555:x64-windows: 726 ms
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+live555
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[live555]+Build+error+on+x64-windows&body=Copy+issue+body+from+D%3A%2FProgramData%2Fcdev%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

--- a/ports/live555/portfile.cmake
+++ b/ports/live555/portfile.cmake
@@ -1,9 +1,9 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "http://live555.com/liveMedia/public/live.2023.11.08.tar.gz"
-    FILENAME "live.2023.11.08.tar.gz"
-    SHA512 b9c0c8a96607325b81e5829b0a57917f155d0767f02a0ee4d887ec7985e5f4a2252961dc4f48de6b66a9ea566fa2a3bea87004c44e565da87b8fa06a3e08954d 
+    URLS "http://live555.com/liveMedia/public/live.2023.11.30.tar.gz"
+    FILENAME "live.2023.11.30.tar.gz"
+    SHA512 c91703197448f65d63a8a6e07597791da1ee63d3b59a0809454468a3869d86e21533c7fc434d0178a179d5a2e0e5614539e47daad813fbbe7468028419026892
 )
 
 vcpkg_extract_source_archive(

--- a/ports/live555/vcpkg.json
+++ b/ports/live555/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "live555",
-  "version-date": "2023-11-08",
+  "version-date": "2023-11-30",
   "description": "A complete RTSP server application",
   "homepage": "http://www.live555.com/liveMedia",
   "license": "GPL-3.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5225,7 +5225,7 @@
       "port-version": 2
     },
     "live555": {
-      "baseline": "2023-11-08",
+      "baseline": "2023-11-30",
       "port-version": 0
     },
     "llfio": {

--- a/versions/l-/live555.json
+++ b/versions/l-/live555.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "80eb597529bc497ceb274509b6224bf0524930c1",
+      "version-date": "2023-11-30",
+      "port-version": 0
+    },
+    {
       "git-tree": "302e545ae4e1386eb3048b429fff813c336ff0c7",
       "version-date": "2023-11-08",
       "port-version": 0

